pkg/contracts/src/CVStrategy.sol

```diff
@@ -167,13 +168,11 @@ contract CVStrategy is BaseStrategy, IPointStrategy, ERC165 {
 
     uint256 public constant PRECISION_SCALE = 10 ** 4;
     uint256 public constant D = 10000000; //10**7
-    uint256 public constant PRECISION_PERCENTAGE = 100 * PRECISION_SCALE;
-    // uint256 public constant ONE_HUNDRED_PERCENT = 1e18;
+    // uint256 public constant PRECISION_PERCENTAGE = 100 * PRECISION_SCALE;
     uint256 private constant TWO_128 = 0x100000000000000000000000000000000; // 2**128
     uint256 private constant TWO_127 = 0x80000000000000000000000000000000; // 2**127
     uint256 private constant TWO_64 = 0x10000000000000000; // 2**64
-    //    uint256 public constant ABSTAIN_PROPOSAL_ID = 1;
-    uint256 public constant MAX_STAKED_PROPOSALS = 10; //@todo not allow stake more than 10 proposals per user
+    uint256 public constant MAX_STAKED_PROPOSALS = 10; //@todo not allow stake more than 10 proposals per user, dont count executed?
```

Comment:
Have you ever think about increasing it? or is too havy in terms of costs for the searchs with the for(....)

Reply:
we dont have it implented that yet, so idk how much that will cost already for 10. also that impact the UI in some way.